app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,12 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreetings } from '../redux/greetings/greetingsSlice';
+
+const Greeting = () => {
+  const greetings = ['Hello', 'Hi', 'Hey', 'Greetings', 'Welcome'];
+  const Greeting = greetings[Math.floor(Math.random() * greetings.length)];
+
+  return <h1>{ Greeting }, World!</h1>;
+}
+
+export default Greeting;
```

Comment:
- [ ] When I Run the React app uses the URL to displays a random greeting I got the following error
- The error message you’re seeing below is related to a missing module in your JavaScript application. The error message indicates that the module ../redux/greetings/greetingsSlice cannot be found, which is required in your Greeting.js component.

![image](https://github.com/DianaBeki/hello-rails-react/assets/93598189/a3e5fb40-14c8-4241-9525-03fe3581a731)